velox/connectors/hive/iceberg/IcebergDataSink.cpp

```diff
@@ -239,6 +344,9 @@ std::vector<std::string> IcebergDataSink::commitMessage() const {
       ("fileFormat", "PARQUET")
       ("content", "DATA");
     // clang-format on
+    if (!(commitPartitionValue_.empty() || commitPartitionValue_[i].isNull())) {
+      commitData["partitionDataJson"] = toJsonString(commitPartitionValue_[i]);
```

Comment:
toJsonString function is used only once and is very short; consider removing it and writing out logic here; this would make it easier to understand the overall format of commit data